commands/curation/curationaudit.go

```diff
@@ -333,7 +352,8 @@ func (ca *CurationAuditCommand) auditTree(tech coreutils.Technology, results map
 	sort.Slice(packagesStatus, func(i, j int) bool {
 		return packagesStatus[i].ParentName < packagesStatus[j].ParentName
 	})
-	results[fmt.Sprintf("%s:%s", projectName, projectVersion)] = packagesStatus
+	results[strings.TrimSuffix(fmt.Sprintf("%s:%s", projectName, projectVersion), ":")] = packagesStatus
+	log.Info(fmt.Sprintf("total time: %v", time.Since(start)))
```

Comment:
```suggestion
	log.Info(fmt.Sprintf("Audit %s dependency tree is done, elapsed time: %v", tech, time.Since(start)))
```
Timing and log should be added outside the func

Reply:
Fixed